jobs/definition/weekly.sh

```diff
@@ -1,4 +1,9 @@
 #!/bin/sh
 node jobs/populate_embeddings.js
 node jobs/clean_cubes.js
-node jobs/download_cubes.js
+rm -r exports/
+node --max-old-space-size=8192 download_drafts.js
+node --max-old-space-size=8192 download_cubes.js
+node --max-old-space-size=8192 download_decks.js
+tar -cJf exports.tar.xz exports/
+aws s3 cp exports.tar.xz s3://cubecobra/exports.tar.xz
```

Comment:
I want to do this via node, not the awscli so it's less dependent on deployment details, such as how the aws profile is configured, instead of using the credentials in `.env`

Reply:
I can add a script that does that last step in node I guess, tar is much better done from the command line though I believe.